dist/settings.json

```diff
@@ -24,6 +22,12 @@
         "ess"
       ]
     },
+  {
+      "CommandName": "Home",
+      "Commands": [
+        "home"
+      ]
+    },
```

Comment:
I would actually prefer if it was added at the bottom of the settings.json file as it makes writing a Change log much easier